scripts/images/cache.yaml

```diff
@@ -0,0 +1,9 @@
+exclude:
+  repos:
+    - docker.io/rancher/harvester
+    - docker.io/rancher/harvester-webhook
+    - docker.io/rancher/harvester-upgrade
+  tags:
+    - master-head
```

Comment:
Very useful enhancement!
Can we support `*-head` ? will be useful for custom branch build

Reply:
Fixed! Images with`*-head` tags are always pulled now.
Users still can add more tags to exclude in the `cache.yaml` file.